synapse/config/cache.py

```diff
@@ -217,9 +221,16 @@ def read_config(self, config, **kwargs) -> None:
                     e.message  # noqa: B306, DependencyException.message is a property
                 )
 
-        expiry_time = cache_config.get("expiry_time")
-        if expiry_time:
-            self.expiry_time_msec: Optional[int] = self.parse_duration(expiry_time)
+        expire_caches = cache_config.get("expire_caches", True)
+        cache_entry_ttl = cache_config.get("cache_entry_ttl")
+
```

Comment:
Since we're changing config options, we should include backwards-compatibility support for the previous config option name. Thus if someone set `expiry_time` previously, that same value should continue to work even after this update. At some point :tm: in the future we can remove support for the old option name.